js/join.js

```diff
@@ -109,9 +109,19 @@ function fetchSavedDetails() {
     .then((res) => {
       window.localStorage.setItem('firstName', res.first_name);
       window.localStorage.setItem('lastName', res.last_name);
-      console.log(res);
+      if (res.statusCode === 401) {
+        alert('You are not logged in! Redirecting you to login.');
+        location.href =
+          'https://github.com/login/oauth/authorize?client_id=23c78f66ab7964e5ef97';
```

Comment:
Nit: Move all of these to a constants file. Maybe this should come from a constants npm package? 🤔